components/service/glean/README.md

```diff
@@ -61,7 +61,8 @@ class SampleApplication : Application() {
 }
 ```
 
-Once initialized, glean will automatically start collecting and sending its [core metrics](metrics.yaml).
+Once initialized, if collection is enabled, glean will automatically start collecting [core metrics](metrics.yaml)
```

Comment:
Shouldn't "core metrics" be "baseline metrics" here?  (I'm always a little unclear about core vs. baseline).

Reply:
Gah, good point. Looks like I'm getting confused too :)